tests/devices/qubit/test_apply_operation.py

```diff
@@ -1267,3 +1274,118 @@ def circuit(init_state):
         results = circuit(tf.Variable(states))
         assert qml.math.shape(results) == (3, 256)
         assert np.array_equal(results[:, 128], [-1.0 + 0.0j] * 3)
+
+
+# pylint: disable=too-few-public-methods
+class TestConditionalsAndMidMeasure:
+    """Test dispatching for mid-circuit measurements and conditionals."""
+
+    @pytest.mark.parametrize("ml_framework", ml_frameworks_list)
```

Comment:
The test will be skipped by CI otherwise.

```suggestion
    @pytest.mark.all_interfaces
    @pytest.mark.parametrize("ml_framework", ml_frameworks_list)
```